apps/web/src/pages/integrations/components/IntegrationInput.tsx

```diff
@@ -73,6 +73,28 @@ export function IntegrationInput({
     );
   }
 
+  if (credential.type === 'boolean') {
+    return (
+      <Switch
+        styles={() => ({
+          root: {
+            display: 'block !important',
+            maxWidth: '100% !important',
+          },
+        })}
+        label={credential.displayName}
+        required={credential.required}
+        placeholder={credential.displayName}
+        description={credential.description ?? ''}
+        data-test-id={credential.key}
+        error={errors[credential.key]?.message}
+        {...register(credential.key)}
+        checked={field.value}
+        onChange={field.onChange}
+      />
+    );
+  }
```

Comment:
This change will make all boolean values for all integrations to become a Switch. We should review all the providers to see nothing breaks in the functionality.
Adding screenshots of how they would look like now would be also helpful for the PR reviewers.

Reply:
yes, although, it makes sense for a boolean to be a toggle rather than an input field. I know I have been confused by that when creating documentation for ([sparkpost](https://docs.novu.co/channels/email/sparkpost/)). 

The only providers affected are Sparkpost and Nodemailer (I checked). The others have a type of switch, which frankly I don't understand why it is so, `boolean` sounds like a better option since it's only true or false. (Note that this switch type is only included in the nodemailer config as well so it's the same behaviour as the boolean.